ChatSecure/Classes/View Controllers/OTRRoomOccupantsViewController.swift

```diff
@@ -123,38 +154,40 @@ open class OTRRoomOccupantsViewController: UIViewController {
         self.navigationController?.navigationBar.setBackgroundImage(self.navigationBarBackground, for: .default)
     }
     
-    open func createHeaderCell(indexPath:IndexPath, type:String) -> UITableViewCell {
+    private func updateMuteUnmuteCell() {
+        var muted = false
+        if let room = self.room {
+            muted = room.isMuted
+        }
+        self.tableHeaderView?.setView(OTRRoomOccupantsViewController.HeaderCellMute, hidden: muted)
+        self.tableHeaderView?.setView(OTRRoomOccupantsViewController.HeaderCellUnmute, hidden: !muted)
+    }
+    
+    open func createHeaderCell(type:String) -> UITableViewCell {
+        var cell:UITableViewCell?
         switch type {
         case OTRRoomOccupantsViewController.HeaderCellGroupName:
-            let cell = tableView.dequeueReusableCell(withIdentifier: type, for: indexPath)
+            cell = tableView.dequeueReusableCell(withIdentifier: type)
```

Comment:
Why no longer using the `indexPath` version? If it's crashing you just need to register the cell/nib for the identifier.

Reply:
Well, we are just using the fact that the tableview owns the designs for the cells and knows how to instantiate them. We are not actually _using_ the cells in the table, we add their views to the header stack view. That's why we don't use the IndexPath. I guess another option would be to create xibs for the different cell types, I just find it convenient to have them all gathered in the tableView in the storyboard.